lib/graphql/index.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+/**
+ * Singleton object for graphql things.
+ */
+
+const graphql = require('graphql');
```

Comment:
Added graphql as dependency. It allows the models to be communicated with by bridge and bridge gui and billing server.